render.go

```diff
@@ -125,6 +126,14 @@ func New(options ...Options) *Render {
 
 	return &r
 }
+//get finally options
+func (r *Render)GetFinallyOptions() Options{
```

Comment:
This isn't used in your code... and I've intentionally kept the options internal. Please remove.